src/main/groovy/org/gradle/android/workarounds/RoomSchemaLocationWorkaround.groovy

```diff
@@ -252,6 +289,15 @@ class RoomSchemaLocationWorkaround implements Workaround {
         }
     }
 
+    private static Closure onlyIfAnnotationProcessorConfiguredForKsp(def commandLineArgumentProviders, Closure<?> action) {
+        return {
+            def provider = commandLineArgumentProviders.get().find { it instanceof KspRoomSchemaLocationArgumentProvider }
```

Comment:
won't this always return true in the cases it is called since we are unconditionally adding this on line 237?